resources/assets/js/features/user/actionTypes.js

```diff
@@ -1,4 +1,13 @@
 export const USER_REGISTER_SUCCESS = 'USER_REGISTER_SUCCESS';
 export const USER_REGISTER_FAILED = 'USER_REGISTER_FAILED';
 export const USER_VERIFY_FAILED = 'USER_VERIFY_FAILED';
-export const USER_VERIFY_SUCCESS= 'USER_VERIFY_SUCCESS';
\ No newline at end of file
+export const USER_VERIFY_SUCCESS= 'USER_VERIFY_SUCCESS';
+
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
```

Comment:
better move all login actions and logic to separate "feature" with own reducer and etc